src/DotLiquid.Website/Views/Home/Index.cshtml

```diff
@@ -12,7 +12,7 @@
     </p>
 
     <h2>Requirements</h2>
-    <p>.NET Framework 4.5 or above.</p>
+    <p>.NET Framework 4.5 or above, .NET Core 2.0 or above, or .NET 5.0 or above.</p>
```

Comment:
This should read: `.NET Framework 4.5 or above, .NET Core 2.0 or above, or .NET 6.0 or above.`

Even though ".NET Core 2.0 or above" includes .NET 6.0, it's ok to state it explicitly.